### PR TITLE
refactor: change to monorepo

### PR DESCRIPTION
**Pull request by nakasyou**
## Description

Recreated #34 

## Checklist

- [ ] Run tests (optional)
- [ ] Add jsdoc (optional)
- [ ] Test in your environment (optional)

If you feel good :), please do the contents of the checklist.

### CODE REVIEW

File: packages/types/deno.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"name": "@evex/linejs-types",
+	"version": "0.0.0",
+	"license": "MIT",
+	"exports": {
+		".": "./line_types.ts",
+		"./entities": "./entities/index.ts",
```
Comment (nakasyou) [inline packages/types/deno.json:7]: https://github.com/evex-dev/linejs/pull/34#discussion_r1759644838

Comment (nakasyou) [reply packages/types/deno.json]: @EdamAme-x changed it.

**Comment by EdamAme-x** [comment]
The link to types in the README is broken.

**Comment by nakasyou** [comment]
> The link to types in the README is broken.

fixed it!

**Review by EdamAme-x — APPROVED** [review]
LGTM

**Pull request merged**